dist/licenses/LICENSE.jboss-marshalling.txt

```diff
@@ -0,0 +1,179 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        https://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+   1. Definitions.
+
+      "License" shall mean the terms and conditions for use, reproduction,
```

Comment:
With a standard AL2 like this, we don't need to add it.

Reply:
The file is referenced in NOTICE-dist.txt (Content of NOTICE for hadoop-auth-2.8.0). So I had to add it.